modules/integration/src/test/scala/scala/cli/integration/util/BloopUtil.scala

```diff
@@ -71,5 +71,15 @@ object BloopUtil {
           args
         )
     }
-
+  def killBloop() = {
+    val javaProcesses = os.proc("jps", "-l").call().out.text().linesIterator
```

Comment:
For later: instead of using `jps`, we could rely on some libdaemon-jvm stuff, that [writes a file with the daemon PID](https://github.com/scala-cli/libdaemon-jvm/blob/965ffeb0328191a453f1a8744038cff04f2265a6/library/src/libdaemonjvm/client/Connect.scala#L25-L31). But its API doesn't provide an easy way to read this PID for now…